### PR TITLE
GOTV block (3/3)

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request does some simplification and clean up for the GOTV block!

### How should this be reviewed?

👀 

### Any background context you want to provide?

We decided against using a toggle in the previous PR #2371 and this is mostly clean up so that it can be QA'd for use starting in October!

### Relevant tickets

References [Pivotal #174219622 ](https://www.pivotaltracker.com/story/show/174219622).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

**Review by Dinnall — APPROVED** [review]
LGTM!

File: resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        />
+      </div>
+
+      <div className="w-full md:w-3/5 flex justify-around py-6 md:py-0">
+        <label htmlFor={`voting-method-in-person-${row}`}>
+          <input
+            id={`voting-method-in-person-${row}`}
+            className="mr-2 md:mr-4"
+            name={`voter-method-${row}`}
+            value={`in-person-${row}`}
```
Comment (aaronschachter) [inline resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js:27]: I think we'd want to save these values as `in-person` and `mail` - is there a reason we need the `row` appended? Later when we save the data and reload it, it'll be a bit easier to check whether the value for row was set to `in-person` or `mail` vs `in-person-2`/`mail-2`

File: resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            type="radio"
+            aria-label="voting-method"
+          />
+          Voting in-person
+        </label>
+        <label htmlFor={`voting-method-mail-${row}`}>
```
Comment (aaronschachter) [inline resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js:33]: Let's add a line break between [siblings here. ](https://github.com/DoSomething/communal-docs/tree/master/Contributing#line-breaks)
```suggestion

        <label htmlFor={`voting-method-mail-${row}`}>
```

File: resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          className="w-full border border-dashed rounded border-gray-600 p-3"
+          type="text"
+          placeholder="Friend's First Name"
+        />
+      </div>
+
+      <div className="w-full md:w-3/5 flex justify-around py-6 md:py-0">
+        <label htmlFor={`voting-method-in-person-${row}`}>
+          <input
+            id={`voting-method-in-person-${row}`}
```
Comment (aaronschachter) [inline resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js:24]: Similar Q to simplifying the names., but do we need an `id` for these at all (I don't think we're using anymore per removing the handleClick for now)

File: resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          type="text"
+          placeholder="Friend's First Name"
+        />
+      </div>
+
+      <div className="w-full md:w-3/5 flex justify-around py-6 md:py-0">
+        <label htmlFor={`voter-method-${row}`}>
+          <input
+            className="mr-2 md:mr-4"
+            name={`voter-method-${row}`}
```
Comment (aaronschachter) [inline resources/assets/components/blocks/GetOutTheVoteBlock/PeopleFormField.js:25]: Nit: Can we rename `voter-method` to `voting-method` for consistency with the `aria-label`, and the [`voting_method` Northstar field](https://github.com/DoSomething/northstar/pull/1062)?

**Review by aaronschachter — APPROVED** [review]
Just one small change, otherwise looks great! 💯 cleanup

**Pull request merged**